src/Microsoft.Extensions.Configuration.Abstractions/IConfiguration.cs

```diff
@@ -36,5 +36,6 @@ public interface IConfiguration
         IEnumerable<IConfigurationSection> GetChildren();
 
         IChangeToken GetReloadToken();
+        IConfigurationRoot Root { get; }
```

Comment:
Minor thing while I'm here, should we make GetReloadToken() into a read only property instead of a method?  Its just returning a field currently...